.yamato/project-tests.yml

```diff
@@ -0,0 +1,29 @@
+{% metadata_file .yamato/project.metafile %}
+---
+
+# For every platform and editor version, run its project tests without
+# running package tests too since they are handled on their respective
+# jobs
+{% for project in projects -%}
+{% for editor in project.test_editors -%}
+{% for platform in test_platforms -%}
+test_{{ project.name }}_{{ editor }}_{{ platform.name }}:
+  name : {{ project.name }} project tests - {{ editor }} on {{ platform.name }}
+  agent:
+    type: {{ platform.type }}
+    image: {{ platform.image }}
+    flavor: {{ platform.flavor}}
+  commands:
+    - npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
+    - pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
+    - unity-downloader-cli -u {{ editor }} -c editor -w --fast
+    - {% if platform.name == "ubuntu" %}DISPLAY=:0 && {% endif %}upm-ci project test -u {{ editor }} --project-path {{ project.path }} --type project-tests
```

Comment:
mm why no display on ubuntu?

Reply:
That seems to be an artifact from the NGO repo. I've checked the docs [here](https://internaldocs.unity.com/package_development/ci/customize-ci/) and that requirement is no longer needed for Ubuntu. However, it's a requirement for centos, if we want to add that as a platform as well.

Reply:
centos is usually used for it's long term support guarantee, for servers. Knowing Multiplay runs ubuntu, there's very little chance we'll touch centos.